README.md

```diff
@@ -2,6 +2,8 @@
 
 Ignition is the utility used by CoreOS Container Linux to manipulate disks during the initramfs. This includes partitioning disks, formatting partitions, writing files (regular files, systemd units, networkd units, etc.), and configuring users. On first boot, Ignition reads its configuration from a source of truth (remote URL, network metadata service, hypervisor bridge, etc.) and applies the configuration.
 
+Ignition has two main development branches: master and spec2x. This is master which is for Ignition included in Red Hat CoreOS and Fedora CoreOS. For Ignition development for Container Linux see the spec2x branch.
```

Comment:
Worthwhile to change the `spec2x branch` at the end into a link?

Reply:
Also might make the wording better to say `This is the master branch`

Reply:
Fixed.